OSPSuite.Presentation.Importer/Presenters/ColumnMappingPresenter.cs

```diff
@@ -101,10 +102,8 @@ public void ChangeUnitsOnRow(ColumnMappingViewModel model)
             {
                column.Unit = unitsEditorPresenter.SelectedUnit;
                model.Description = ColumnMappingFormatter.Stringify(model.Source);
-               OnDataFormatParametersChanged?.Invoke(this, new DataFormatParametersChangedArgs()
-               {
-                  Parameters = new List<DataFormatParameter>() { activeRow.Source }
-               });
+               _view.RefreshList();
```

Comment:
THis is the one method that is needed. This is defacto very similar to what you were doing before with the begin update and endupdate. We need to rebing to the lists.